teaching/index.md

```diff
@@ -81,7 +81,7 @@ Julia is now being used in several universities and online courses. If you know
   * [CS51400](https://www.cs.purdue.edu/homes/dgleich/cs514-2016/), Numerical Analysis (Prof. [David Gleich](https://www.cs.purdue.edu/homes/dgleich/))
 
 - University of Edinburgh, Spring 2016
-  * [MATH11146](http://www.drps.ed.ac.uk/15-16/dpt/cxmath11146.htm), Modern optimization methods for big data problems (Prof. [Peter Richtarik](http://www.maths.ed.ac.uk/~prichtar/index.html))
+  * [MATH11146](http://www.drps.ed.ac.uk/16-17/dpt/cxmath11146.htm), Modern optimization methods for big data problems (Prof. [Peter Richtarik](http://www.maths.ed.ac.uk/~prichtar/index.html))
```

Comment:
This is under Spring 2016 here but the new link refers to the academic year 2016-2017.